Makefile

```diff
@@ -229,38 +227,15 @@ rm -rf $${tmpdir} ;\
 endef
 
 ##### Proto #####
-$(PROTO_OUT):
-	@mkdir -p $(PROTO_OUT)
-
-# We depend on gomodtidy to ensure that go.mod is up to date before we delete the generated files
-# in case protogen fails, and we need to rerun it.
-clean-proto: gomodtidy
-	@rm -rf $(PROTO_OUT)/*
-
 $(API_BINPB): go.mod go.sum $(PROTO_FILES)
 	@printf $(COLOR) "Generate api.binpb..."
 	@./cmd/tools/getproto/run.sh --out $@
 
-protoc: clean-proto $(PROTO_OUT) $(PROTOGEN) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRPC) $(PROTOC_GEN_GO_HELPERS) $(API_BINPB)
-	@$(PROTOGEN) \
-		--descriptor_set_in=$(API_BINPB) \
-		--root=proto/internal \
-		--rewrite-enum=BuildId_State:BuildId \
-		-p go-grpc_out=$(PROTO_PATHS) \
-		-p go-helpers_out=$(PROTO_PATHS)
-	@mv -f "$(PROTO_OUT)/temporal/server/api/"* "$(PROTO_OUT)"
-
-# All gRPC generated service files paths relative to PROTO_OUT.
-PROTO_GRPC_SERVICES = $(patsubst $(PROTO_OUT)/%,%,$(shell find $(PROTO_OUT) -name "service.pb.go" -o -name "service_grpc.pb.go"))
-service_name = $(firstword $(subst /, ,$(1)))
-mock_file_name = $(call service_name,$(1))mock/$(subst $(call service_name,$(1))/,,$(1:go=mock.go))
-
-proto-mocks: protoc $(MOCKGEN)
-	@printf $(COLOR) "Generate proto mocks..."
-	$(foreach PROTO_GRPC_SERVICE,$(PROTO_GRPC_SERVICES),\
-		@cd $(PROTO_OUT) && \
-		$(ROOT)/$(MOCKGEN) -copyright_file ../LICENSE -package $(call service_name,$(PROTO_GRPC_SERVICE))mock -source $(PROTO_GRPC_SERVICE) -destination $(call mock_file_name,$(PROTO_GRPC_SERVICE)) \
-	$(NEWLINE))
+protoc: $(PROTOGEN) $(MOCKGEN) $(GOIMPORTS) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRPC) $(PROTOC_GEN_GO_HELPERS) $(API_BINPB)
+	@env \
```

Comment:
Btw there's an all-Makefile version in the previous commit on the PR. I think I generally prefer separate scripts since they're easier to read/maintain, but if you would rather it be all in the Makefile I can do that.

Also do you know any make trick to export a bunch of variables for one command that's cleaner than this? I looked and couldn't find any